po/r-novice-gapminder.ja.po

```diff
@@ -18582,10 +18558,9 @@ msgid ""
 "can then spread our `gap_long()` to the original intermediate format or the\n"
 "widest format. Let's start with the intermediate format."
 msgstr ""
-"It is always good to check work. So, let's use the opposite of `gather()` to\n"
-"spread our observation variables back out with the aptly named `spread()`. We\n"
-"can then spread our `gap_long()` to the original intermediate format or the\n"
-"widest format. Let's start with the intermediate format."
+"仕事を確認することは、とても良いことです。 `gather()` の反対の名は体を表す `spread()` を使って、\n"
+"観測変数を元通りに広げてみましょう。そこから `gap_long()` を、元々の中間形式、又は、\n"
+"一番横長な形式に広げることもできます。まずは、中間形式から始めましょう。"
```

Comment:
Original has been updated. https://swcarpentry.github.io/r-novice-gapminder/14-tidyr/index.html

Reply:
leave as-is for now